lib/looker.ts

```diff
@@ -0,0 +1,53 @@
+import { IDashboardElement, IWriteQuery } from "@looker/sdk"
+import { SDK } from "./sdk";
+
+export async function getAWDashboardElement0(): Promise<IDashboardElement> {
+  const dashboardId = "1471";
+
+  const elements: IDashboardElement[] = await SDK.ok(SDK.dashboard_dashboard_elements(dashboardId));
```

Comment:
Add an XXX comment here for the performance work: since we're only getting one element, we should be able to switch this out for the more performant `dashboard_element`.  https://mozilla.cloud.looker.com/extensions/marketplace_extension_api_explorer::api-explorer/4.0/methods/Dashboard/dashboard_element has more info.